includes/widgets/class-wc-widget-rating-filter.php

```diff
@@ -98,28 +98,29 @@ public function widget( $args, $instance ) {
 		ob_start();
 
 		$found         = false;
-		$rating_filter = isset( $_GET['rating_filter'] ) ? array_filter( array_map( 'absint', explode( ',', wp_unslash( $_GET['rating_filter'] ) ) ) ) : array(); // WPCS: input var ok, CSRF ok, sanitization ok.
+		$rating_filter = isset( $_GET['rating_filter'] ) ? absint( wp_unslash( $_GET['rating_filter'] ) ) : ''; // WPCS: input var ok, CSRF ok, sanitization ok.
```

Comment:
There could potentially be multiple ratings in the filter, e.g. `http://local.wordpress.test/shop/?orderby=rating&rating_filter=2,3`. You have made it so that this won't work properly in that case by removing all the logic that handles that.

Reply:
Hi @claudiulodro, I thought that. But this previous multiple rating filter is kinda weird cause- 

**Firstly,** I think when people clicks on 3 star rating they wanna see the the products with 3 star, NOT all the products with 3 star, 2 star and 1 star.

**Secondly,** may be if they select all the 3 star, 2 star and 1 star all the products with these rating can be shown together. But from previous code it was creating a link for each stars like below table-

| Star  | Rating Filter Value |
| ------------- | ------------- | 
|  1  | 1             |
|  2  | 2, 1             |
|  3  | 3, 2, 1         |
|  4  | 4, 3, 2, 1     |
|  5  | 5, 4, 3, 2, 1 | 

Which is not right I think. If there were a system where people can select multiple star rating then this way we can manage things like if they select 2 star and 1 star then we can pass `rating_filter` value `2,1`.

Reply:
@claudiulodro Need your feedback on this.

Reply:
Sorry, I didn't notice this comment. I think the existing way is correct, and we should keep it. It lets you add and remove ratings from the filters easily. I think changing that behavior is outside the scope of the bug fix, and we'll likely get complaints if we do so.